resources/web/map/js/trajectory_raw.js

```diff
@@ -159,12 +160,13 @@ function syncLayers() {
  * @param points optional number of points to render ellipse, higher for smoothness
```

Comment:
you need to move this doc string to be an inline comment about the `let points = 16; line.` OR undo the refactor of the optional param to a function.